modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js

```diff
@@ -487,7 +487,7 @@
 					const eventData = event.data;
 
 					let file = eventData.file;
-					const image = eventData.el.$;
+					const image = eventData.el?.$ || eventData.element?.$;
```

Comment:
It should be safe to just replace `el` with `element`. In both places where we fire `imageAdd`, [first](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js#L240-L243) and [second](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js#L298-L303), we are setting `element`.